### PR TITLE
(PA-4902) Add puppet8 nightly release packages

### PR DESCRIPTION
**Pull request by AriaXLi**

### CODE REVIEW

File: source/nightlies.puppet.com/puppet8-nightly-release.repo.template [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[puppet8-nightly]
+name=Puppet 8 Nightly Repository __OS_NAME__ __OS_VERSION__ - $basearch
+baseurl=http://nightlies.puppet.com/yum/puppet8-nightly/__OS_NAME__/__OS_VERSION__/$basearch
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-2025-04-06-puppet8-nightly-release
+       file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet8-nightly-release
```
Comment (e-gris) [inline source/nightlies.puppet.com/puppet8-nightly-release.repo.template:5]: This line is obsolete and can be removed.

File: source/projects/puppet8-nightly-release.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "project": "puppet8-nightly-release",
+  "description": "Release packages for the Puppet 8 nightly repository",
+  "version": "1.0.0",
+  "release": "1",
+
+  "license": "ASL 2.0",
+  "vendor": "Puppet, Inc. <release@puppet.com>",
```
Comment (e-gris) [inline source/projects/puppet8-nightly-release.json:8]: This should now be `Puppet By Perforce <release@puppet.com>`

**Review by e-gris — CHANGES_REQUESTED** [review]
A couple of tiny changes. Otherwise LGTM.

File: source/nightlies.puppet.com/puppet8-nightly-release.repo.template (hunk 1 above)
Comment (AriaXLi) [reply source/nightlies.puppet.com/puppet8-nightly-release.repo.template:5]: Ah okay just to verify, lines 2-5 should be removed?

**Review by e-gris — APPROVED** [review]

**Pull request merged**